src/cpp/session/resources/dependencies/r-packages.json

```diff
@@ -596,22 +533,8 @@
         "tutorial": {
             "description": "RStudio Tutorials",
             "packages": [
-                "checkmate",
-                "ellipsis",
-                "evaluate",
-                "fastmap",
-                "htmltools",
-                "htmlwidgets",
-                "jsonlite",
-                "knitr",
                 "learnr",
```

Comment:
Shouldn't we keep learnr's dependencies here as well?

Reply:
✅